client/post-editor/post-editor.jsx

```diff
@@ -299,7 +301,8 @@ export class PostEditor extends React.Component {
 				<EditorPostTypeUnsupported />
 				<EditorForbidden />
 				<EditorRevisionsDialog loadRevision={ this.loadRevision } />
-				<EditorGutenbergDialogs />
+				{ isEnabled( 'editor/deprecation-notice' ) && <EditorDeprecationDialog /> }
+				{ ! isEnabled( 'editor/deprecation-notice' ) && <EditorGutenbergDialogs /> }
```

Comment:
This change is what breaks the learn more button. I think we leave this line how it is.